lib/ansible/modules/files/file.py

```diff
@@ -49,7 +49,6 @@
         If C(touch) (new in 1.4), an empty file will be created if the C(path) does not
         exist, while an existing file or directory will receive updated file access and
         modification times (similar to the way `touch` works from the command line).
-    default: file
```

Comment:
While the argspec does not specify file as the default, it is the default if `path` does not exist. Not sure if we should remove it, if we do remove it then this should be added to the description of state

Reply:
Yes you're right, I think I will simply rollback this since the case was already managed by test/sanity/validate-modules/ignore.txt

Reply:
rollbacked